db/articles.js

```diff
@@ -0,0 +1,59 @@
+let articleList = {
+  articles: []
+};
+
+let urlTitle = null;
+
+//article POST function
+function create(obj) {
+  obj.urlTitle = obj.title.replace(/ /g, "-");
+  articleList.articles.push(obj);
+  return articleList.articles;
+}
+
+function getArt(paramTitle) {
+  let formattedTitle = paramTitle
+    .split(" ")
+    .join("")
+    .toLowerCase();
+  let lengthOfArticles = articleList.articles.length;
+  for (let i = 0; i < lengthOfArticles; i++) {
+    if (
+      articleList.articles[i].title
+        .split(" ")
+        .join("")
+        .toLowerCase() === formattedTitle
+    ) {
```

Comment:
You should be using the urlTitle to compare rather than reformatting the title to compare